src/Notify/Notify.csproj

```diff
@@ -3,7 +3,7 @@
   <PropertyGroup>
     <RestoreProjectStyle>PackageReference</RestoreProjectStyle>
     <TargetFrameworks>netstandard2.0;netcoreapp2.0;net462</TargetFrameworks>
-    <Version>2.2.0</Version>
+    <Version>2.2.1</Version>
```

Comment:
This should be a minor update rather than a patch:

`2.3.0`

Reply:
Updated